services/gratipay/gratipay.service.js

```diff
@@ -12,14 +12,14 @@ module.exports = [
   deprecatedService({
     route: {
       base: 'gittip',
-      format: '(?:/user|/team|/project)?/(?:.*)',
+      pattern: ':various?',
```

Comment:
I'm in the process of trying to use this in another service. Should this be `:various*`?

Reply:
elsewhere yeah it probably should be. the existing test route just used the base `gratipay` so i went with `?` to keep the tests passing without _completely_ dropping the other stuff. Probably doesn't matter all that much for a deprecated service anyway though 😄

Reply:
Hehe it's true. Though I still see a lot of those "no longer available" badges still on readmes!

I think `:various*` (or `:various+`) would match e.g. `/user/foobar` like the old regex. I'll open a PR to update it.